src/misc/fetch-meta.ts

```diff
@@ -2,6 +2,7 @@ import Meta, { IMeta } from '../models/meta';
 
 const defaultMeta: any = {
 	name: 'Misskey',
+	uri: '',
```

Comment:
What else could it be?
It should never be able to fall back to this default because of https://github.com/syuilo/misskey/blob/8f714b5b126226346af850337566f0f70bd02d4f/src/config/load.ts#L30

config.url should always exist